src/stories/containers/cookies-policy/cookies-policy-banner.mvvm.ts

```diff
@@ -1,18 +1,24 @@
 import { useCallback, useMemo, useState } from 'react';
 import { useCookies } from 'react-cookie';
 import { useScrollLock } from '../../../core/hooks/scroll-hooks';
-import { useMediaQuery } from '@mui/material';
+interface Props {
+  timestampTracking: string;
+  themeTracking: string;
+  analyticsTracking: string;
```

Comment:
here you should reuse the same type declared at the `_app.ts`, maybe move it to a type file like the `types_utils` or some other place where it can be reuser more easily as this type can be used in several places

Reply:
done